infra/docker-compose/jupyter/startup.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# Clone Feast repository into Jupyter container
+git clone -b ${FEAST_REPOSITORY_VERSION} --single-branch https://github.com/gojek/feast.git
```

Comment:
This will fail on multiple `docker-compose up` as Feast repo already exists.
We should add a test for the `feast` directory an not clone feast if it already exists.

Reply:
Do you mean if they reconfigure `jupyter` and then want to restart it? Running `docker-compose up` twice doesnt restart the container. To be safe we can add `|| true` to ensure that the script doesnt fail here

Reply:
The workflow that would fail is:
- `docker-compose up` - when they first try out Feast's docker-compose setup.
- `docker-compose down` - after they are trying out.
- `docker-compose up` - when they to use it again. The failure happens here.

Reply:
Good point.

Reply:
Actually, I can't produce this error using those commands. It clones successfully every time.

Reply:
The exact commands i used to create the error:
- `docker-compose -f docker-compose.yml -f docker-compose.online.yml up`
- Ctrl-C to bring down docker-compose stack
- `docker-compose -f docker-compose.yml -f docker-compose.online.yml up`

This is the error I got:
```
jupyter_1         | + git clone -b v0.4.7 --single-branch https://github.com/gojek/feast.git
jupyter_1         | fatal: destination path 'feast' already exists and is not an empty directory.
feast_jupyter_1 exited with code 128
```

Reply:
Ok, "works" on my side. Added the change.